src/main/java/net/neoforged/neoform/runtime/actions/ApplySourceTransformAction.java

```diff
@@ -124,7 +143,37 @@ public void run(ProcessingEnvironment environment) throws IOException, Interrupt
         Collections.addAll(args, "{input}", "{output}");
         setArgs(args);
 
-        super.run(environment);
+        try {
+            super.run(environment);
+        } catch (Exception e) {
+            // Pass through *all* problems if possible
+            try {
+                environment.getProblemReporter().tryMergeFromFile(problemsReport);
+            } catch (IOException ex) {
+                LOG.warn("Failed to pass-through problem report from " + problemsReport + ": " + e);
+            }
+
+            throw e;
+        }
+
+        // Pass through any relevant problems to the outer problem context
+        if (Files.exists(problemsReport)) {
+            var problems = FileProblemReporter.loadRecords(problemsReport);
+            for (var problem : problems) {
+                if (problem.location() == null || validatedAccessTransformers.contains(problem.location().file())) {
```

Comment:
I would be careful with file paths here. I remember AT doing some normalization. Maybe this is more of a JST issue than an NFRT issue (JST should return paths using the same format that it was given, ideally).

Reply:
I think we need the paths to be absolute. Not sure if picocli handles it for us?